chaos.py

```diff
@@ -37,7 +25,7 @@ def main():
 
     log = logging.getLogger("chaosbot")
 
-    api = gh.API(settings.GITHUB_USER, settings.GITHUB_SECRET)
+    gh.API(settings.GITHUB_USER, settings.GITHUB_SECRET)
```

Comment:
[Creating an instance of API][0] doesn't have a side effect, so there's no point in doing so if the result isn't used. There's another API instance [here][1] which is (as far as I see), the only one used. This can be done away with completely.

[0]: https://github.com/chaosbot/Chaos/blob/c2498c4621a5f890bd704d568a9192acd9ab5711/github_api/__init__.py#L33-L36
[1]: https://github.com/chaosbot/Chaos/blob/c2498c4621a5f890bd704d568a9192acd9ab5711/cron/poll_pull_requests.py#L18